src/Controller/UI/DatasetSubmissionController.php

```diff
@@ -318,7 +318,6 @@ protected function newDatasetFile(DatasetSubmission $datasetSubmission)
     {
         $datasetSubmission->setDatasetFileTransferStatus(DatasetSubmission::TRANSFER_STATUS_NONE);
```

Comment:
This is also broken, this function is gone now?

Reply:
No, we din't merge that branch. let me test again